packages/checkout-extension/src/withExtension.ts

```diff
@@ -0,0 +1,7 @@
+import { createInjectHoc } from '@bigcommerce/checkout/locale';
```

Comment:
Another comment not related to this PR importantly is that we need to get this `createInjectHoc` out of locale package. Maybe as a followup after this.

Reply:
Noted. Shall I create a ticket?

Reply:
Yeah I think good idea, maybe with the onboarding label 😄

Reply:
For the record, the ticket is https://bigcommercecloud.atlassian.net/browse/CHECKOUT-7633.